internal/resources/metal/project_ssh_key/resource.go

```diff
@@ -184,8 +186,8 @@ func (r *Resource) Delete(
 	id := state.ID.ValueString()
 
 	// Use API client to delete the resource
-	deleteResp, err := client.SSHKeys.Delete(id)
-	if equinix_errors.IgnoreResponseErrors(equinix_errors.HttpForbidden, equinix_errors.HttpNotFound)(deleteResp, err) != nil {
+	deleteResp, err := client.SSHKeysApi.DeleteSSHKey(context.Background(), id).Execute()
+	if equinix_errors.IgnoreMetalResponseErrors(equinix_errors.HttpForbidden, equinix_errors.HttpNotFound)(deleteResp, err) != nil {
```

Comment:
This (and other references to the new function) should use the pre-existing `equinix_errors.IgnoreHttpResponseErrors` function